src/style.css

```diff
@@ -0,0 +1,152 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
don't apply this for `*`

Reply:
Shall I apply "padding 0, margin 0"  for  ".searchContainer--big  * " or "form *" ?

Reply:
padding: 0   for   inputs  will be enough.